mathcomp/Make

```diff
@@ -80,6 +80,8 @@ ssreflect/fingraph.v
 ssreflect/finset.v
 ssreflect/fintype.v
 ssreflect/generic_quotient.v
+ssreflect/monoid.v
+ssreflect/comoid.v
```

Comment:
Except if we expect them to rapidly grow a lot, we could probably consider merging the two files.

Reply:
I do not really expect them to grow, it should have the structures most people may be looking for. There are two structures I know I will add in `monoid.v` and a few more instances, but that is about it. I am fine with merging, but we would get another 2k lines file

Reply:
It of course depends on compilation time but I'd say anything <= 3k lines is ok.
It also depends the expected use: if lot of users want to use monoid without comoid then the spliting makes sense but if virtually anyone using monoid also uses comoid then the spliting makes much less sense. So it depends your expected use of the files.